### PR TITLE
Test combinations of attribute types

### PR DESCRIPTION
**Pull request by hajoeichler**
@svenmueller please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ This component allows you to process given _CSV_ files in order to generate SPHE
 
 ```bash
 $ npm install -g sphere-product-type-json-generator
-
+```
+* build javascript sources
```
Comment (svenmueller) [inline README.md]: i think this should be in a seperate section called "Development". only in this case you will need to build the js files

**Comment by svenmueller** [comment]
@hajoeichler are the attribute files used somewhere?

**Comment by hajoeichler** [comment]
@svenmueller thx to @stmeissner we now have many combinations.
I have integrated the first bunch in the cucumber test and they already produce a lot of API errors.

Please have a look at them.

**Comment by hajoeichler** [comment]
@svenmueller file already gone at https://github.com/sphereio/sphere-product-type-json-generator/commit/de2c3834e05836f4b441a30840f5825179bd0a74

**Pull request merged**